bazaarvoice/emodb
# migrate from Objects.firstNonNull()

## Github Issue #

None

## What Are We Doing Here?

- replace guava `Objects.firstNonNull()` to java8 analogues
- fix style/formatting.

## How to Test and Verify

1. Check out this PR
2. Run `mvn clear verify`
3. Profit

## Risk

### Level 

`Low`
### Required Testing

`Smoke`, `Regression`, or `Manual`.

### Risk Summary

As changes mostly contains refactoring but it's still possible to have some typo.

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
